netlify/transDimConfig.js

```diff
@@ -0,0 +1,34 @@
+const transDimConfig = {
+   // SMTP SETTINGS of server that will handle the actual sending of the emails
+   smtp: {
+      host: process.env.SMTP_HOST,
+      port: process.env.SMTP_PORT,
+      secure: process.env.SMTP_SECURE == "true",
+      require_tls: process.env.SMTP_REQUIRE_TLS = "true",
+      auth_user: process.env.SMTP_USERNAME,
+      auth_pass: process.env.SMTP_PASSWORD
+   },
+   // ALLOWED DOMAIN is the domain the form is hosted on that will be sending the requests
+   allow_domain: "*", // allows requests only from this domain - IMPORTANT! use '*' (wildcard) as the value for testing only
+   // The root URL of your deployed function
+   netlify_deploy_url: "https://fervent-colden-5f0cd2.netlify.app", // the base URL, without an ending / (forward slash)
+   // EMAIL STYLING of the generated email
+   email_styling: {
+      header_text: "Request to join The Trans Dimension", // a title, keep as short as possible
+      header_background_colour: "FF7AA7", // hex codes only, not including the #
+      header_text_color: "040F39", // hex codes only, not including the #
+      body_text_color: "040F39", // hex codes only, not including the #
+      footer_background_colour: "53C3FF", // hex codes only, not including the #
+      footer_text_color: "040F39", // hex codes only, not including the #
+      netlify_header_image: "", // this has to be deployed to Netlify - if not in the root, add a path, e.g. "images/logo.jpeg" - without a starting / (forward slash)
```

Comment:
If I understand purpose (branding), then blank is fine here - because we are only sending to internal folks at the moment. Might be worth a comment?